net/remote/remote-node.sh

```diff
@@ -242,6 +243,11 @@ EOF
         --keypair ~/solana/config/bootstrap-leader/identity-keypair.json \
         validator-info publish "$(hostname)" -n team/solana --force || true
     fi
+
+    if [[ -n "$netemConfig" ]]; then
```

Comment:
Looks like this should live in ~/solana/on-reboot so that the config gets applied if the instance is rebooted